.eslintrc

```diff
@@ -2,7 +2,39 @@
     "extends": "girder",
     "rules": {
         "no-throw-literal": "off",
+        "backbone/defaults-on-top": "error",
```

Comment:
You may have answered this already, but what is the rational behind putting these rules here rather than in our eslint config repo?